lib/generator.js

```diff
@@ -548,7 +550,8 @@ class Generator {
   async generateFile(asyncapiDocument, fileName, baseDir) {
     const sourceFile = path.resolve(baseDir, fileName);
     const relativeSourceFile = path.relative(this.templateContentDir, sourceFile);
-    const targetFile = path.resolve(this.targetDir, relativeSourceFile);
+    const targetFile = path.resolve(this.targetDir, this.maybeRenameSourceFile(relativeSourceFile));
+    console.log(targetFile);
```

Comment:
remove this console.log

Reply:
LOL thanks!

Reply:
BTW, this would be a great use case for the debug param. I want to see the name of the target file but of course I don't want to see the rest of the log messages.